pyproject.toml

```diff
@@ -61,6 +59,7 @@ prometheus-async = {extras = ["aiohttp"], version = "^22.1.0"}
 autoagora-isa = {git = "ssh://git@github.com/semiotic-ai/autoagora-isa.git", rev = "08142a5dcf3cd44c957921d67792d5188d4894ea", optional = true}
 backoff = "^2.2.1"
 python-json-logger = "^2.0.4"
+autoagora-agents = {git = "https://github.com/semiotic-ai/autoagora-agents.git"}
```

Comment:
hmmm, do I remember correctly you wanted to depend on a particular release?
should we build autoagora-agents wheels?

Reply:
Indeed. Did it there: https://github.com/semiotic-ai/autoagora/pull/20/commits/667fcadaec04d16880fd1711df63b9e25b74e0ed